concepticondata/conceptlists/Gerardi-2020-244.tsv

```diff
@@ -0,0 +1,245 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	PORTUGUESE
+Gerardi-2020-244-1	1	ALL	98	ALL	TODOS / TUDO
+Gerardi-2020-244-2	2	ARM	1673	ARM	BRAÇO
+Gerardi-2020-244-3	3	ARROW	977	ARROW	FLECHA
+Gerardi-2020-244-4	4	ASH	646	ASH	CINZA(S)
+Gerardi-2020-244-5	5	AXE	677	AXE	MACHADO
+Gerardi-2020-244-6	6	BACK	1291	BACK	COSTAS
+Gerardi-2020-244-7	7	BAD/EVIL	3764	BAD OR EVIL	MAU / MAL / RUIM
+Gerardi-2020-244-8	8	BAMBOO	1927	BAMBOO	TAQUARA / BAMBU
+Gerardi-2020-244-9	9	BANANA	868	BANANA	BANANA
+Gerardi-2020-244-10	10	BASKET	1539	BASKET	CESTO / CESTA
+Gerardi-2020-244-11	11	BAT	1793	BAT	MORCEGO
+Gerardi-2020-244-12	12	BE_WRONG	1390	WRONG	ERRAR / ESTAR ERRADO
+Gerardi-2020-244-13	13	BEANS	2574	BEAN OR PEA	FEIJÃO
+Gerardi-2020-244-14	14	BEARD	678	BEARD	BARBA
+Gerardi-2020-244-15	15	BELLY	1838	NAVEL	UMBIGO
+Gerardi-2020-244-16	16	BREAK	680	BREAK (DESTROY OR GET DESTROYED)	QUEBRAR / ROMPER
+Gerardi-2020-244-17	17	BIG	1202	BIG	GRANDE
+Gerardi-2020-244-18	18	BIRD	937	BIRD	PÁSSARO
+Gerardi-2020-244-19	19	BITE	1403	BITE	MORDER
+Gerardi-2020-244-20	20	BLACK	163	BLACK	PRETO
+Gerardi-2020-244-21	21	BLOW	176	BLOW (WITH MOUTH)	SOPRAR
+Gerardi-2020-244-22	22	BONE	1394	BONE	OSSO
+Gerardi-2020-244-23	23	BOY	1366	BOY	MENINO
+Gerardi-2020-244-24	24	BRANCH	1531	BRANCH	GALHO
+Gerardi-2020-244-25	25	BREAST	1402	BREAST	SEIO
+Gerardi-2020-244-26	26	BREATHE	1407	BREATHE	RESPIRAR
+Gerardi-2020-244-27	27	BURN_intr	2102	BURN	QUEIMAR
+Gerardi-2020-244-28	28	BURN_tr	141	BURN (SOMETHING)	QUEIMAR
+Gerardi-2020-244-29	29	BURY	1719	BURY	ENTERRAR
+Gerardi-2020-244-30	30	BUSH	201	FARM	ROÇA
```

Comment:
The English gloss does not fit.

Reply:
This is the author's mapping, a bit off indeed. @LanguageStructure , I see from the data this is sometimes cognate with Old Tupi ka'a, correct? Do you think FOREST 420 would be more appropriate?